src/sonic-dhcp-utilities/dhcp_utilities/dhcpservd/dhcp_cfggen.py

```diff
@@ -107,7 +107,7 @@ def _parse_dpu(self, dpus_table, mid_plane_table):
         Returns:
             Parsed obj, sample:
                 mid_plane = {
-                    "bridge": "bridge_midplane",
+                    "bridge": "bridge-midplane",
```

Comment:
Is it a valid requirement for the alignment? Does any guidance mentioned the naming convention?

Reply:
@qiluo-msft It is as per the HLD. Please refer to https://github.com/sonic-net/SONiC/pull/1586/files